src/main/java/org/generationcp/middleware/domain/germplasm/GermplasmDto.java

```diff
@@ -34,106 +34,106 @@
 
 	private List<GermplasmNameDto> names;
 
-	private List<GermplasmAttributeDto> attributes;
+	private List<GermplasmAttributeRequestDto> attributes;
```

Comment:
why is this class being replaced? to me makes more sense to use GermplasmAttributeDto when building a new GermplasmDto